lib/fathead/react_native/output.txt

```diff
@@ -0,0 +1,1410 @@
+DatePickerIOS.timeZoneOffsetInMinutes	A							http://facebook.github.io/react-native/docs/getting-started.html			<section class="prog__container"><p>Timezone offset in minutes.</p><pre><code>timeZoneOffsetInMinutes number</code></pre></section>	https://facebook.github.io/react-native/releases/0.40/docs/datepickerios.html#timezoneoffsetinminutes
+Dimensions	A							http://facebook.github.io/react-native/docs/getting-started.html			<section class="prog__container"><p>This should only be called from native code by sending the didUpdateDimensions event.</p></section>	https://facebook.github.io/react-native/releases/0.40/docs/dimensions.html#
+Dimensions Dimensions.get	R	Dimensions.Dimensions.get										
+Dimensions Dimensions.set	R	Dimensions.Dimensions.set										
+Dimensions.Dimensions.get	A							http://facebook.github.io/react-native/docs/getting-started.html			<section class="prog__container"><p>Initial dimensions are set before runApplication is called so they should be available before any other require's are run, but may be updated later.</p><pre><code>get(dim)</code></pre></section>	https://facebook.github.io/react-native/releases/0.40/docs/dimensions.html#get
+Dimensions.Dimensions.set	A							http://facebook.github.io/react-native/docs/getting-started.html			<section class="prog__container"><p>This should only be called from native code by sending the didUpdateDimensions event.</p><pre><code>set(dims)</code></pre></section>	https://facebook.github.io/react-native/releases/0.40/docs/dimensions.html#set
+Direct Manipulation	A							http://facebook.github.io/react-native/docs/getting-started.html			<section class="prog__container"><p>It is sometimes necessary to make changes directly to a component without using state/props to trigger a re-render of the entire subtree. When using React in the browser for example, you sometimes need to directly modify a DOM node, and the same is true for views in mobile apps. setNativeProps is the React Native equivalent to setting properties directly on a DOM node.</p></section>	https://facebook.github.io/react-native/releases/0.40/docs/direct-manipulation.html#
+DrawerLayoutAndroid	A							http://facebook.github.io/react-native/docs/getting-started.html			<section class="prog__container"><p>React component that wraps the platform DrawerLayout (Android only). The Drawer (typically used for navigation) is rendered with renderNavigationView and direct children are the main view (where your content goes). The navigation view is initially not visible on the screen, but can be pulled in from the side of the window specified by the drawerPosition prop and its width can be set by the drawerWidth prop.</p></section>	https://facebook.github.io/react-native/releases/0.40/docs/drawerlayoutandroid.html#
+DrawerLayoutAndroid DrawerLayoutAndroid.closeDrawer	R	DrawerLayoutAndroid.DrawerLayoutAndroid.closeDrawer										
```

Comment:
Looking at https://developer.android.com/reference/android/support/v4/widget/DrawerLayout.html it seems like the duplication of the class name isn't right. Let's remove it 👍